librz/bin/bobj.c

```diff
@@ -1017,3 +1017,105 @@ RZ_API ut64 rz_bin_object_v2p(RzBinObject *obj, ut64 vaddr) {
 	}
 	return m->paddr + delta;
 }
+
+/**
+ * \brief   Allocates and initializes the RzBinStrDb structure with the given list of strings
+ *
+ * \param   list  The list of strings to initialize the database with
+ *
+ * \return  On success returns a valid pointer, otherwise NULL
+ */
+RZ_API RZ_OWN RzBinStrDb *rz_bin_string_database_new(RZ_NULLABLE RzList /*<RzBinString *>*/ *list) {
```

Comment:
No, borrow can also mean an argument's ownership is kept at the caller.

Reply:
sorry, i meant RZ_OWN :DDD

Reply:
these cases are very important to notify and they are actually one of the very reasons we introduced RZ_OWN and RZ_BORROW, because otherwise people have no way to know (without reading all the code) that the list they are providing as argument is not theirs anymore.

Reply:
yes it was

Reply:
![image](https://user-images.githubusercontent.com/561184/176732294-3b424700-3f45-4019-95ca-fed7f305dd57.png)